src/ui/simulator/windows/inspector/accumulator.hxx

```diff
@@ -947,12 +947,12 @@ struct PAreaSpilledEnergyCost
 };
 
 // ----------------
-// THERMAL CLUSTERS
+// THERMAL/RENEWABLE CLUSTERS
```

Comment:
ok, but on line 991 starts the **THERMAL CLUSTERS** block, we should have a commented title on that line

Reply:
Done. I created 2 new sections : one for renewables, one for binding constraints.